functions.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```

Comment:
Can you delete this file altogether? Remember, we made a new branch so this branch wouldn't include this file.